app/scripts/services/builds.js

```diff
@@ -42,7 +42,7 @@ angular.module("openshiftConsole")
       var buildType = isJenkinsPipelineStrategy(buildConfig) ? 'pipeline' : 'build';
       var req = {
         kind: "BuildRequest",
-        apiVersion: "v1",
+        apiVersion: buildConfig.apiVersion,
```

Comment:
This should be the `apiVersion` built from `buildConfigsInstantiateVersion`. It's not necessarily the same version as `buildConfig.apiVersion`

Reply:
So that is interesting because if I change it to:

```JavaScript 
apiVersion: buildConfigsInstantiateVersion.version,
// which is this:
// {group: "build.openshift.io", version: "v1", resource: "buildconfigs/instantiate"}
```
it will error:

```
apiVersion: "v1"
code: 400
kind: "Status"
message: "the API version in the data (v1) does not match the expected API version (build.openshift.io/v1)"
metadata: {}
reason: "BadRequest"
status: "Failure"
```
So we need a helper fn to concat `buildConfigInstantiateVersion.group + '/' + buildConfigInstantiateVersion.version` to ensure these are correct?